kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue

```diff
@@ -400,8 +436,13 @@
       },
     },
     created() {
-      BookmarksResource.fetchCollection().then(bookmarks => {
-        this.bookmarksCount = bookmarks.length;
+      ContentNodeResource.fetchBookmarks({
+        params: { limit: 25, kind: ContentNodeKinds.EXERCISE },
```

Comment:
Oh, looks like we need to fetch bookmarked topics that contain exercises as well as exercises themselves?

I think we've not actually implemented topic bookmarking yet, right? So maybe we can simplify the 'topic exercises' code above with that in mind?

Reply:
That's a good point -- yes right now there isn't any topic bookmarking, so will update